content/docs/reference/cmctl.md

```diff
@@ -348,10 +348,26 @@ $ cmctl upgrade migrate-api-version --qps 5 --burst 10
 
 ## Legacy kubectl plugin
 
-While the kubectl plugin is supported, it is recommended to use `cmctl` as this enables a better experience via tab auto-completion.
+We call the kubectl plugin "legacy" because we used to recommend using the standalone binary
```

Comment:
Can we now stop calling it "legacy" since autocompletion is fixed?

Reply:
Done